jobs/config.json

```diff
@@ -755,6 +755,7 @@
     "args": [
       "--check-leaked-resources",
       "--env-file=jobs/platform/gce.env",
+      "--env=GCE_GLBC_IMAGE=gcr.io/k8s-ingress-image-push/ingress-gce-e2e-glbc-amd64:latest",
```

Comment:
@krzyzacy @BenTheElder One difference I see with the ci-ingress-gce-e2e job is that it does not set the env-file var.
Is it fine to set both env-file and env flags here or should I now remove env-file?

cc @MrHohn

Reply:
@MrHohn is OOO. @rramkumar1 might be able to help.

Reply:
I would use both. We should probably change ci-ingress-gce-e2e to use both as well.

Reply:
What you have here, looks good to me.